src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -722,10 +722,13 @@ def set(self, vol_name: str, subvol_name: str, size: str):
 
         return f'Subvolume {subvol_name} updated successfully'
 
-    def delete(self, vol_name: str, subvol_name: str):
+    def delete(self, vol_name: str, subvol_name: str, retain_snapshots: bool = False):
+        params = {'vol_name': vol_name, 'sub_name': subvol_name}
+        retain_snapshots = str_to_bool(retain_snapshots)
+        if retain_snapshots:
```

Comment:
```suggestion
        if str_to_bool(retain_snapshots):
```

Reply:
Wouldn't you want to keep it as a bool? Since we are explicitly declaring it as a bool and that's what you would expect from this variable?